libnopegl/src/backends/vk/rendertarget_vk.c

```diff
@@ -381,8 +383,10 @@ void ngli_rendertarget_vk_freep(struct rendertarget **sp)
     vkDestroyRenderPass(vk->device, s_priv->render_pass, NULL);
     vkDestroyFramebuffer(vk->device, s_priv->framebuffer, NULL);
 
-    for (uint32_t i = 0; i < s_priv->nb_attachments; i++)
+    for (uint32_t i = 0; i < s_priv->nb_attachments; i++) {
+        NGLI_RC_UNREFP(&s_priv->attachments_refs[i]);
         vkDestroyImageView(vk->device, s_priv->attachments[i], NULL);
```

Comment:
It looks like of safe but just in case, should the unrefp be after the destroy of the image view (otherwise the image view would be left with dangling reference to the destroyed attachment)

Reply:
Done.